openedx/core/djangoapps/schedules/templates/schedules/edx_ace/verifiedupgradedeadlinereminder/email/body.html

```diff
@@ -1,7 +1,67 @@
-Dear {{ user_personal_address }},
-<br/>
-We hope you are enjoying {{ course_title }}.
-Upgrade by {{ user_schedule_verified_upgrade_deadline_time|date:"l, F dS, Y" }}
-to get a shareable certificate!
-<br/>
-<a href="{{course_verified_upgrade_url}}">Upgrade now</a>
+{% extends 'schedules/edx_ace/common/base_body.html' %}
+{% load i18n %}
+
+{% block preview_text %}
+    {% if courses|length > 1 %}
```

Comment:
All of these conditionals need `courses` to be replaced with `course_id_strs`

Reply:
wait, `course_ids` right? https://github.com/edx/edx-platform/blob/3a52f28c94fbdac20ff77692dfa402a98b45a6c7/openedx/core/djangoapps/schedules/tasks.py#L323